Obedient-Lobsters/BangazonAPI
# Creation of CustomerPayment Model

# Description
 Created CustomerPayment Model for intersection table between Customer #1  and PaymentTypes  #3 
 Fixes # (#1 , #3 )
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions
 -Ensure that Customer.cs and PaymentType.cs is included in the Model folder

-In the directory BangazonAPI, use command "dotnet run" to run program. The server should then start and run without errors.

-Make sure all names line up with ERD.(https://drive.google.com/file/d/1Qo9O2oUezNd13Wsr0_VUbUtwwDo4-oAt/view)
 # Checklist:
